depthai_nodes/ml/messages/creators/image.py

```diff
@@ -51,3 +51,14 @@ def create_image_message(
     imgFrame.setType(img_frame_type)
 
     return imgFrame
+
+
+def getVisualizationMessage(self) -> dai.ImgFrame:
+    img_frame = dai.ImgFrame()
+    mask = self._map.copy()
```

Comment:
What is the `self` in this case? This function looks like a copy of the one in `Map2D` type.

Reply:
I must have copied it from Map2D by mistake. It's removed now.